### DISCUSSION TITLE
Is the API rate limit for a given operation shared or independent among different spawned instances of an authorized application?

### DISCUSSION THREAD

**Discussion by mpenna** [amzn/selling-partner-api-models#3935] [General]
### Discussed in https://github.com/amzn/selling-partner-api-docs/discussions/3512

<div type='discussions-op-text'>

<sup>Originally posted by **mpenna** December 28, 2023</sup>
I have an application where eventually many client instances spawned with the same app credentials (client id, secret, etc.) but having each their own auth tokens created through the use of the same seller's refresh token could be requesting data concurrently from the same operation endpoint. In such a situation, will they all be considered in aggregate towards said operation's burst and rate limit specifications or will each instance be individually considered for such usage?</div>

**Comment by hexrw** [comment]
It is shared. I work with this API often and from what I've observed the throttling happens based on the combination of your client id + your seller's id

> [All usage plans are specific to application-Selling Partner pairs, so that your throughput grows naturally with your clients.](https://developer-docs.amazon.com/sp-api/docs/usage-plans-and-rate-limits-in-the-sp-api#will-my-application-be-throttled-more-often-as-i-obtain-more-authorizations)
